DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.13.04.SqlDataProvider

```diff
@@ -0,0 +1,36 @@
+/************************************************************/
+/*****              SqlDataProvider                     *****/
+/*****                                                  *****/
+/*****                                                  *****/
+/***** Note: To manually execute this script you must   *****/
+/*****       perform a search and replace operation     *****/
+/*****       for {databaseOwner} and {objectQualifier}  *****/
+/*****                                                  *****/
+/************************************************************/
+
+/*    Insert Detailed Permissions                           */
+/************************************************************/
+
+INSERT INTO {databaseOwner}[{objectQualifier}Permission] ([PermissionCode], [ModuleDefID], [PermissionKey], [PermissionName], [ViewOrder], [CreatedByUserID], [CreatedOnDate], [LastModifiedByUserID], [LastModifiedOnDate]) VALUES ('SYSTEM_FOLDER', -1, 'ADD', 'Add', 1, -1, GETDATE(), -1, GETDATE())
```

Comment:
Should these be protected inserts to avoid duplication?

Reply:
Yes, I will add protection to avoid duplicates.

Reply:
You may use SQL merge statement instead of insert.